hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -357,6 +363,58 @@
         }
       ]
     },
+    { name: "EXTHT_HI_THRESHOLDS",
+      desc: "External health test high thresholds register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "REGEN",
+      fields: [
+        { bits: "15:0",
+          name: "FIPS_EXTHT_HI_THRESH",
+          desc: '''This is the threshold size for the external health test.
+                   This is value used in normal mode when entropy is being tested in
```

Comment:
is value -> value is

Reply:
thanks for reviewing @msfschaffner! - as always, thanks for finding these annoying typos. This was copied from another spot in error, fixed all in this file.